helm/mongodb/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+annotations:
+  category: Database
+  licenses: Apache-2.0
+apiVersion: v2
+appVersion: 6.0.8
+dependencies:
+  - name: common
+    repository: oci://registry-1.docker.io/bitnamicharts
+    tags:
+      - bitnami-common
+    version: 2.x.x
+description: MongoDB(R) is a relational open source NoSQL database. Easy to use, it stores data in JSON-like documents. Automated scalability and high-performance. Ideal for developing cloud native applications.
+home: https://bitnami.com
+icon: https://bitnami.com/assets/stacks/mongodb/img/mongodb-stack-220x234.png
+keywords:
+  - mongodb
+  - database
+  - nosql
+  - cluster
+  - replicaset
+  - replication
+maintainers:
+  - name: VMware, Inc.
+    url: https://github.com/bitnami/charts
+name: mongodb
+sources:
+  - https://github.com/bitnami/charts/tree/main/bitnami/mongodb
```

Comment:
Curious, did you select this Helm chart for a particular reason?

Reply:
We need mongodb for grip to work. I believe https://github.com/bitnami/charts/tree/main/bitnami/mongodb is the official helm chart for mongodb, but I could very well be wrong about that

Reply:
Okay sounds good, no need to revisit until we do testing but will leave this comment open

Reply:
Yeah. An issue could be made from that. Not sure which one is better but that one that I'm currently using works.